templates/content/oe-theme-helper-in-page-navigation--node--oe-organisation.html.twig

```diff
@@ -0,0 +1,9 @@
+{#
+/**
+ * @file
+ * Theme override to display in-page navigation for Organisation nodes.
+ *
+ * @see ./modules/oe_theme_helper/templates/oe-theme-helper-in-page-navigation.html.twig
+ */
+#}
+{% include "oe-theme-helper-in-page-navigation--with-image.html.twig" %}
```

Comment:
Use extends instead. Otherwise other templates can't pass variables and blocks further than this.